### PR TITLE
Fixes a trivial bug in Idenitity.cs found by code scan

### PR DESCRIPTION
**Pull request by whoisj**

### CODE REVIEW

File: LibGit2Sharp/Identity.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public Identity(string name, string email)
             Ensure.ArgumentNotNullOrEmptyString(name, "name");
             Ensure.ArgumentNotNullOrEmptyString(email, "email");
             Ensure.ArgumentDoesNotContainZeroByte(name, "name");
-            Ensure.ArgumentDoesNotContainZeroByte(name, "email");
+            Ensure.ArgumentDoesNotContainZeroByte(email, "email");
```
Comment (Therzok) [inline LibGit2Sharp/Identity.cs]: dammit. DAMMIT. People could've crashed!

Good job coverity!

**Comment by shiftkey** [comment]
:sparkles:

**Comment by whoisj** [comment]
Coverity missed this one. Found by manual code scan.

**Comment by nulltoken** [comment]
:confounded:

**Comment by nulltoken** [comment]
@whoisj :+1:

**Pull request merged**